app/data_model/answer_store.py

```diff
@@ -4,23 +4,27 @@
 
 
 class Answer(object):
-    def __init__(self, answer_id, block_id, value, answer_instance=0):
+    def __init__(self, answer_id, block_id, group_id, value, answer_instance=0, group_instance=0):
```

Comment:
This is only a small point, but semantically, does it make sense to order these in the same order that we'd expect them to appear in the schema e.g. group, block, answer?